contracts/cw20/Cargo.toml

```diff
@@ -13,12 +13,19 @@ backtraces = ["cosmwasm-std/backtraces"]
 library = []
 
 [dependencies]
-cosmwasm-schema = { workspace = true }
+{% if workspace == "false" %}cosmwasm-schema = "=1.1.9"
+cosmwasm-std = "=1.1.9"
+cw-utils = "=1.0.1"
+cw2 = "=1.1.0"
+cw20 = "=1.1.0"
+cw-storage-plus = "=1.1.0"
+{% else %}cosmwasm-schema = { workspace = true }
+cosmwasm-std = { workspace = true }
```

Comment:
How about using cargo-generate condition grammer and Rhai extensions?
https://cargo-generate.github.io/cargo-generate/templates/scripting.mini-example.html

Reply:
I'll try it.